src/ledger/NetworkConfig.cpp

```diff
@@ -1303,6 +1329,7 @@ SorobanNetworkConfig::loadFromLedger(AbstractLedgerTxn& ltxRoot,
     {
         loadParallelComputeConfig(ltx);
         loadLedgerCostExtConfig(ltx);
+        loadSCPTimingConfig(ltx);
```

Comment:
nit: could we use p23 in the if condition for this load? `PARALLEL_SOROBAN_PHASE_PROTOCOL_VERSION` is a bit confusing, since the SCP configs are unrelated